app.vue

```diff
@@ -1,10 +1,93 @@
+<script>
+export default {
+	mounted() {
+		if (localStorage.getItem('color-theme') === 'dark' ||
+			(!('color-theme' in localStorage) &&
+				window.matchMedia('(prefers-color-scheme: dark)').matches)) {
+			document.documentElement.classList.add('dark');
+		} else {
+			document.documentElement.classList.remove('dark');
+		}
+	},
+	data() {
+		return {
+			levelNumber: this.levelNumber = useRoute().fullPath.split('/')[2],
+			levelCount: 20
+		}
+	},
+	methods: {
+		toggleTheme() {
+			if (localStorage.getItem('color-theme')) {
+				if (localStorage.getItem('color-theme') === 'light') {
+					document.documentElement.classList.add('dark');
+					document.querySelectorAll('NuxtLink').forEach((el) => el.classList.add('dark'));
+					localStorage.setItem('color-theme', 'dark');
+				} else {
+					document.documentElement.classList.remove('dark');
+					document.querySelectorAll('NuxtLink').forEach((el) => el.classList.remove('dark'));
+					localStorage.setItem('color-theme', 'light');
+				}
+
+			} else {
+				if (document.documentElement.classList.contains('dark')) {
+					document.documentElement.classList.remove('dark');
+					localStorage.setItem('color-theme', 'light');
+				} else {
+					document.documentElement.classList.add('dark');
+					localStorage.setItem('color-theme', 'dark');
+				}
+			}
+		},
+	},
+};
+</script>
+
 <template>
-  <div class="flex h-svh max-h-svh flex-col bg-bg-light dark:bg-bg-dark">
-    <header class="grid h-24 flex-none place-items-center">
-      <p class="text-black dark:text-white">Header</p>
-    </header>
-    <main class="h-0 flex-1">
-      <NuxtPage />
-    </main>
-  </div>
+	<div class="flex flex-col dark:bg-bg-dark bg-bg-light h-screen">
+		<header class="flex justify-between w-11/12 h-20 mt-6 mx-auto mb-6 items-center">
+			<div class="pr-9 flex items-center">
+				<img src="~/assets/img/accessolotl.svg" onerror="this.src='~/assets/img/accessolotl.png'"
+					class="h-11 dark:hidden">
+				<img src="~/assets/img/icons/dark/accessolotllogo-dark.svg"
+					onerror="this.src='~/assets/img/icons/dark/accessolotllogo-dark.png'"
+					class="h-11 hidden dark:block">
+			</div>
+			<div class="flex grow items-center dark:bg-levelbar-dark bg-levelbar-light h-16 rounded-2xl">
+				<button type="button" class=" ml-12 invisible">
+					<Icon name="material-symbols:wb-sunny-outline" class="text-4xl" />
+				</button>
+				<nav class="flex justify-center items-center align-center flex-grow">
+					<NuxtLink :to="`${parseInt(levelNumber) - 1}`" @click="levelNumber--" class="mr-2">
+						<Icon name="material-symbols:arrow-back-ios-rounded"
+							class="dark:text-t-dark text-t-light text-2xl" />
+					</NuxtLink>
+					<span class="text-2xl dark:text-t-dark text-t-light">Level {{ levelNumber }} of {{ levelCount
+						}}</span>
+					<NuxtLink :to="`${parseInt(levelNumber) + 1}`" @click="levelNumber++" class="ml-2">
```

Comment:
Bitte das `:to=...` in eine computed property und das `@click` in eine Funktion, wäre sauberer